packages/xo-server-netbox/src/index.js

```diff
@@ -233,6 +239,107 @@ class Netbox {
 
     log.info(`Synchronizing ${xoPools.length} pools with Netbox`, { pools: xoPools })
 
+    // Tenants -----------------------------------------------------------------
+
+    let nbTenants
```

Comment:
The code is starting to be hard to follow, I think before the next change, we should split this function.